posthog/models/filters/filter.py

```diff
@@ -36,7 +37,207 @@
 from posthog.utils import relative_date_parse
 
 
-class Filter(PropertyMixin):
+class DisplayMixin:
+    _data: Dict
```

Comment:
Noticing we're repeating this every mixin - maybe we can set up a mixin for the mixins? :D

Reply:
yup done